internal/aiptest/create/create_time.go

```diff
@@ -35,8 +35,8 @@ var createTime = suite.Test{
 		f.P(ident.AssertNilError, "(t, err)")
 		f.P(ident.AssertCheck, "(t, msg.CreateTime != nil)")
 		f.P(ident.AssertCheck, "(t, !msg.CreateTime.AsTime().IsZero())")
-		// Allow Created == Now due to flakyness of clock in podman and colima
-		f.P(ident.AssertCheck, "(t, !msg.CreateTime.AsTime().After(", ident.TimeNow, "()))")
+		// Allow Created == Now+1s due to flakyness of clock in podman and colima
+		f.P(ident.AssertCheck, "(t, !msg.CreateTime.AsTime().After(", ident.TimeNow, "().Add(1 * time.Second)))")
```

Comment:
random thought - since this is only flaky on local machines running on MacOS, how about we generate an if statement that gates the more relaxed check to only run if `runtime.GOOS == "darwin"`? that way we can still have a stricter check in the CI/CD pipelines?

Reply:
Yes, very good idea. 👍 💯